src/Analyzer.Utilities/BasicSyntaxGeneratorExtensions.cs

```diff
@@ -1,34 +1,32 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.Composition;
-using System.Diagnostics;
+﻿using System.Diagnostics;
```

Comment:
This is one of those Git confusions. I removed CSharpOverloadOperatorEqualsOnOverridingValueTypeEquals.Fixer.cs and BasicOverloadOperatorEqualsOnOverridingValueTypeEquals.Fixer.vb, and I created CSharpSyntaxGeneratorExtensions.cs and BasicSyntaxGeneratorExtensions.cs. Git decided that I had renamed CSharpOverloadOperatorEqualsOnOverridingValueTypeEquals.Fixer.cs to BasicSyntaxGeneratorExtensions.cs.